src/Components/test/testassets/BasicTestApp/FileDownloaderComponent.razor

```diff
@@ -0,0 +1,32 @@
+@using Microsoft.AspNetCore.Components
+
+@inject IFileDownloader FileDownload
+
+<h1>File configuration</h1>
+File name:
+<br />
+
+<input type="text" @bind-value="@fileName" />
+<br />
+
+<h1>File download</h1>
+
+<button @onclick="SaveFile"> Save </button>
+
+@code {
+    private string fileName = "file.txt";
+
+    private Stream GetFileStream()
+    {
+        byte[] Data = new byte[] { 1, 2, 3, 4, 5, 6, 7, 8, 9 };
+        var fileStream = new MemoryStream(Data);
```

Comment:
Unfortunately, I don't believe this'll be sufficient as it's less than 32kb (SignalR default message size). We should try something like:

```suggestion
        var data = new byte[100 * 1024];
        for (var i = 0; i < data.Length; i++)
        {
            data[i] = (byte)(i % 256);
        }
        var fileStream = new MemoryStream(Data);
```

We'll also have to add a separate test for `SaveAs` but using a `byte[]`. It should just be a matter of copying over the streaming implementation (but max `byte[]` length should be below 32kb due to the aforementioned limit).

Reply:
Then again, we have all this testing in place for the existing streaming/byte[] interop, so maybe this isn't necessary here 🤔 

Thoughts on this?